luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -215,6 +216,31 @@ def hgmd(self):
                             'class': self._hgmd_data[self.mt.row_key].info.CLASS})
 
 
+
+    @row_annotation(disable_index=True)
+    def gnomad_non_coding_constraint(self):
+        return hl.struct(
+            **{
+                "z_score": self._interval_ref_data.index(
+                    self.mt.locus, all_matches=True
+                )
+                .filter(
+                    lambda x: hl.is_defined(x.gnomad_non_coding_constraint["z_score"])
+                )
+                .gnomad_non_coding_constraint.z_score
+            }
+        )
+
+    @row_annotation(disable_index=True)
```

Comment:
in the seqr ticket for this it says we want the ability to filter on these fields, so I think we should not disable indexing

Reply:
Thanks for the reminder, Anne confirmed she wants to search on zscore as well so I enabled indexing there as well.